backend/tests/v5/helper/services.js

```diff
@@ -73,14 +73,14 @@ db.createUser = (userCredentials, tsList = [], customData = {}) => {
 
 db.createTeamspaceRole = (ts) => createTeamspaceRole(ts);
 
-// breaking = create a broken schema for teamspace to trigger errors for testing
-db.createTeamspace = async (teamspace, admins = [], breaking = false, customData) => {
-	const permissions = admins.map((adminUser) => ({ user: adminUser, permissions: TEAMSPACE_ADMIN }));
+db.createTeamspace = async (teamspace, admins = [], members = [], customData) => {
 	await ServiceHelper.db.createTeamspaceRole(teamspace);
-	return Promise.all([
-		ServiceHelper.db.createUser({ user: teamspace, password: teamspace }, [teamspace],
-			{ permissions: breaking ? undefined : permissions, ...customData }),
-		createTeamspaceSettings(teamspace),
+	await createTeamspaceSettings(teamspace);
+	await ServiceHelper.db.createUser({ user: teamspace, password: teamspace }, [teamspace], customData);
+
+	await Promise.all([
+		...admins.map((adminUser) => grantAdminToUser(teamspace, adminUser)),
+		...members.map((memberUser) => grantMemberToUser(teamspace, memberUser)),
```

Comment:
Very confused how you get this to work. This isn't how we store if the user is a member of the teamspace.

Reply:
actually looking at the references no one ever pass in the data.

So it's added but never used? Just remove it?